.env.development.local

```diff
@@ -0,0 +1,3 @@
+DATABASE_HOSTNAME=psql
+DATABASE_USERNAME=postgres
+DATABASE_PASSWORD=qwerty
```

Comment:
You need to setup your code editor

<img width="247" alt="Screenshot 2023-02-19 at 00 36 54" src="https://user-images.githubusercontent.com/496713/219900535-a898f94d-93e8-474c-8b53-e47a3ba54058.png">